recipes/criterion/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+from os import path
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.meson import MesonToolchain, Meson
+from conan.tools.files import get, apply_conandata_patches, export_conandata_patches, rmdir, copy, collect_libs
+from conan.tools.layout import basic_layout
+from conan.tools.gnu import PkgConfigDeps
+
+
+class CriterionConan(ConanFile):
+    name = "criterion"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Snaipe/Criterion"
+    topics = ("testing")
+    description = "A cross-platform C and C++ unit testing framework for the 21st century"
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    requires = ("libgit2/1.5.0", "libffi/3.4.3")
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.rm_safe("fPIC")
+
+    def configure(self):        
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        del self.settings.compiler.libcxx
```

Comment:
no all compilers have this setting so its needed for 2.0

```suggestion
       self.settings.compiler.rm_safe("libcxx")
```